frontend/src/authProviders/default.ts

```diff
@@ -0,0 +1,16 @@
+import { AuthProvider } from 'src/@types/auth';
+
+export const defaultAuthProvider: AuthProvider = {
```

Comment:
Just curious, what's the purpose of defaultAuthProvider?

Reply:
skeletal implementation of auth provider, used as a fallback.

Reply:
on second thought, we can make jwtauthprovider the fallback instead